scripts/asic_config_check

```diff
@@ -17,14 +17,16 @@ function debug()
     logger -p user.info "$@"
 }
 
-# Retrieve the source ASIC config checksum from the source image
-# Exits with error SRC_ASIC_CONFIG_NOT_FOUND if the checksum is not found
-function GetSourceASICConfigChecksum()
+# Retrieve the current ASIC config checksum from the current image
+# Returns CURR_ASIC_CONFIG_NOT_FOUND if the checksum is not found, ASIC_CONFIG_UNCHANGED otherwise
+function GetCurrentASICConfigChecksum()
 {
     if [[ ! -f "/etc/sonic/asic_config_checksum" ]]; then
-            error "ASIC config not found in src image, can't verify changes"
-            exit "${SRC_ASIC_CONFIG_NOT_FOUND}"
+            debug "ASIC config not found in curr image, pre-check not supported"
+            return "${CURR_ASIC_CONFIG_NOT_FOUND}"
```

Comment:
This return code need to change to unchanged or add code in warm-reboot to allow this code.